src/application-container/private/skip-to-links/SkipToLink.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import uuidv4 from 'uuid/v4';
+
+import NavigationItemContext from '../../../navigation-item/NavigationItemContext';
+
+import SkipToLinksContext from './SkipToLinksContext';
+
+const propTypes = {
+  /**
+   * The string description of the SkipToLink. The provided value will be
+   * automatically prefixed with the string "Skip to ".
+   */
+  description: PropTypes.string.isRequired,
+  /**
+   * The function to be executed upon selection of the SkipToLink.
+   */
+  onSelect: PropTypes.func.isRequired,
+  /**
+   * The priority value indicating the presentation position of the SkipToLink
+   * relative to other registered links.
+   */
+  priority: PropTypes.oneOf(['main', 'other']),
+};
+
+const defaultProps = {
+  priority: 'other',
+};
+
+const SkipToLink = ({
+  description, priority, onSelect,
+}) => {
+  const skipToLinksContext = React.useContext(SkipToLinksContext);
+  const navigationItemContext = React.useContext(NavigationItemContext);
+
+  // A unique identifier is generated for each SkipToLink instance for
+  // registration purposes.
+  const keyRef = React.useRef(uuidv4());
+
+  React.useLayoutEffect(() => {
+    if (!navigationItemContext.isActive) {
```

Comment:
this is nitpicky, but switching the conditional to a truthy check on isActive is slightly more readable
```suggestion
    if (navigationItemContext.isActive) {
      register()
    } else {
      unregister()
```